src/components/task-plan/reading-stats.cjsx

```diff
@@ -0,0 +1,117 @@
+React = require 'react'
+_ = require 'underscore'
+BS = require 'react-bootstrap'
+Router = require 'react-router'
+
+{TaskPlanStore, TaskPlanActions} = require '../../flux/task-plan'
+
+Stats = React.createClass
+  mixins: [Router.State, Router.Navigation]
+
+  getInitialState: ->
+    id = @getParams().id
+    if (id)
```

Comment:
I think the reading stats only apply to an existing reading so the following code is never executed